### PR TITLE
Adding a copy feature without jaggery.tools jar

### PR DESCRIPTION
**Pull request by Sarangan0219**
## Purpose

Resolves https://github.com/wso2/product-apim/issues/8915

Remove `org.jaggeryjs.jaggery.tools_0.14.12.jar` from `org.jaggeryjs.server.feature`  feature and a add new feature of  `org.jaggeryjs.jaggery.tools.server.feature` to add the removed jar.

### CODE REVIEW

File: features/org.jaggeryjs.server.feature/pom.xml [hunk 1]
```diff
@@ -25,10 +25,9 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>org.jaggeryjs.server.feature</artifactId>
     <packaging>pom</packaging>
-    <name>Jaggery Server Feature</name>
+    <name>Jaggery Server Feature (without tools dependency)</name>
```
Comment (ruks) [inline features/org.jaggeryjs.server.feature/pom.xml:28]: we will use the original name and description

File: features/org.jaggeryjs.jaggery.tools.server.feature/pom.xml [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+            <artifactId>org.wso2.carbon.registry.indexing</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.apache.solr</groupId>
+            <artifactId>solr</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jaggeryjs</groupId>
+            <artifactId>org.jaggeryjs.jaggery.tools</artifactId>
+        </dependency>
```
Comment (ruks) [inline features/org.jaggeryjs.jaggery.tools.server.feature/pom.xml:68]: we need only this(org.jaggeryjs.jaggery.tools) dependency and can remove others. So this feature contain only this bundle.

File: features/org.jaggeryjs.server.feature/pom.xml (hunk 1 above)
Comment (Sarangan0219) [reply features/org.jaggeryjs.server.feature/pom.xml]: https://github.com/wso2/jaggery/pull/213/commits/6795bbb5ffaa89c1cf9ad7b904fbc1148bfc73a7

File: features/org.jaggeryjs.jaggery.tools.server.feature/pom.xml (hunk 2 above)
Comment (Sarangan0219) [reply features/org.jaggeryjs.jaggery.tools.server.feature/pom.xml:36]: https://github.com/wso2/jaggery/pull/213/commits/6795bbb5ffaa89c1cf9ad7b904fbc1148bfc73a7

**Comment by Sarangan0219** [comment]
This fix should go to the 0.12.x branch . Hence Closing this PR .
New PR opened in 0.12.x branch (https://github.com/wso2/jaggery/pull/214)

**Pull request closed without merging**